ql/pricingengines/vanilla/exponentialfittinghestonengine.cpp

```diff
@@ -236,22 +238,23 @@ namespace QuantLib {
         const Real sigma = model_->sigma();
         const Real rho   = model_->rho();
 
+        QL_REQUIRE(cv_ == ControlVariate::Gatheral
+            || ControlVariate::BranchCorrection,
+            "Gather and Branch-Correction are not supported as control-variate");
```

Comment:
Should this be `!=` ?  Also, after the `||` there's no condition, just an enum (which is probably why this passes)